swap/swap.go

```diff
@@ -40,10 +40,11 @@ import (
 )
 
 const (
-	deployRetries               = 5
-	deployDelay                 = 1 * time.Second // delay between retries
-	defaultCashInDelay          = uint64(0)       // Default timeout until cashing in cheques is possible - TODO: deliberate value, experiment // should be non-zero once we implement waivers
-	DefaultInitialDepositAmount = 0               // TODO: deliberate value for now; needs experimentation
+	deployRetries                     = 5
+	deployDelay                       = 1 * time.Second // delay between retries
+	defaultCashInDelay                = uint64(0)       // Default timeout until cashing in cheques is possible - TODO: deliberate value, experiment // should be non-zero once we implement waivers
+	DefaultInitialDepositAmount       = 0               // TODO: deliberate value for now; needs experimentation
+	DefaultHarddepositTimeoutDuration = 86400           // this is the amount of time in seconds which an issuer has to wait to decrease the harddeposit of a beneficiary. The smart-contract allows for setting this variable differently per beneficiary
```

Comment:
Please use a `time.Duration` for time spans, as discussed yesterday. See `deployDelay` above as an example. In this case it would be `24 * time.Hours` - then, you convert those to `big.Int` when you call the smart contract